hashintel/hash
# Add Toggle List Item block

## 🌟 What is the purpose of this PR?

Adds a toggle list item block to expand our block collection as part of block week.

## 🔗 Related links

- [Notion doc](https://www.notion.so/Toggle-list-item-show-hide-details-a0333b2fa3d14a7eae2bb4e777aead5b) _(internal)_

## 🔍 What does this change?

- Adds a toggle list item block to the blocks folder.

## ❓ How to test this?

1. Start it as any other block
2. Make sure there are placeholder, both for the title and content
3. Open/close the block and make sure it's persisted
4. Make sure the title and content are persisted
5. Try to add multiple lines to the content, it should grow and shrink based the amount of text

## 📹 Demo

![image](https://user-images.githubusercontent.com/45027362/165739813-905d1730-c193-4851-af2b-3542ca54a491.png)
